src/librustdoc/lib.rs

```diff
@@ -404,8 +404,11 @@ fn main_args(args: &[String]) -> isize {
                                   `short` (instead was `{}`)", arg));
         }
     };
+    let treat_err_as_bug = matches.opt_strs("Z").iter().any(|x| {
+        *x == "treat-err-as-bug"
+    });
```

Comment:
This branch includes your `-Z treat-err-as-bug` branch as well. Please remove that commit from this branch.

Reply:
Woups, indeed!